### PR TITLE
xxHash lib added

### PR DESCRIPTION
**Pull request by newnon**

### CODE REVIEW

File: src/xxhash.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+define $(PKG)_UPDATE
+    $(call MXE_GET_GITHUB_TAGS, Cyan4973/xxHash) | \
+    $(SED) 's,^v,,g'
+endef
+
+define $(PKG)_BUILD
+    mkdir '$(1).build'
+    cd '$(1).build' && cmake . \
+        -DCMAKE_TOOLCHAIN_FILE='$(CMAKE_TOOLCHAIN_FILE)' \
```
Comment (starius) [inline src/xxhash.mk]: Could you replace `cmake -DCMAKE_TOOLCHAIN_FILE='$(CMAKE_TOOLCHAIN_FILE)'` with `'$(TARGET)-cmake'`, please?

File: src/xxhash.mk [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        -DBUILD_STATIC_LIBS=$(CMAKE_STATIC_BOOL) \
+        -DBUILD_SHARED_LIBS=$(CMAKE_SHARED_BOOL) \
+        '$(1)/cmake_unofficial'
+    $(MAKE) -C '$(1).build' -j '$(JOBS)' install VERBOSE=1
```
Comment (starius) [inline src/xxhash.mk]: Could you separate build and install steps, please?
Install step is done with `-j 1` and build step with `-j $(JOBS)`.

File: src/xxhash.mk [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+endef
+
+$(PKG)_BUILD_SHARED =
```
Comment (starius) [inline src/xxhash.mk]: Please run the following command:
```
$ echo >> src/xxhash.mk
```
to append missing `0x0a` byte.

**Comment by newnon** [comment]
Done

**Comment by starius** [comment]
Fixed, rebased and merged manually: https://github.com/mxe/mxe/compare/9ebb730d86dfc9ea416368b2b9042297e3eff9d6...fc78b0c43263ecf13af3186accb8611267cb4a3c

Thanks!

**Comment by starius** [comment]
The patch was sent to upstream: https://github.com/Cyan4973/xxHash/pull/73

**Pull request closed without merging**